### DISCUSSION TITLE
How to optimize the slow startup of minio when there is a large amount of stored data？

### DISCUSSION THREAD

**Discussion by sampson17471** [minio/minio#20771] [Q&A] [answered]
When Minio has a large amount of stored data, it starts very slowly.approximately 200G, it takes an hour to start up. Is there any good way to optimize this? In addition, when storing Minio, besides the data uploaded by users, there are also some other data. What are these data and what is their function?

**Comment by harshavardhana** [comment]
There is no MinIO deployment that takes an hour to start, this looks like have some external factors.

Explain your deployment please..

**Reply by sampson17471** [reply]
I deployed it through k8s with 3 pod nodes，The complete deployment configuration can refer to https://github.com/bitnami/charts/tree/main/bitnami/minio

**Comment by marktheunissen** [comment]
@sampson17471 Please paste all the logs with their timestamps that show you the startup time? Just paste as much information as you can see that makes you believe it takes an hour to startup.

**Reply by klauspost** [reply]
Furthermore, you can set `_MINIO_SERVER_DEBUG=on` - and furthermore be sure you are on latest version.

**Comment by harshavardhana** [comment, marked as answer]
I think this is related to your persistent volume which re applies your fsgroup permissions

And we don't support bitnami MinIO, please deploy MinIO Operator which is the supported version.
